tests/test_dataclass_factory.py

```diff
@@ -4,12 +4,18 @@
 from typing import Callable, Dict, List, Optional, Set, Tuple
 from unittest.mock import ANY
 
+import pytest
 from pydantic.dataclasses import Field  # type: ignore
 from pydantic.dataclasses import dataclass as pydantic_dataclass
 
 from polyfactory.factories import DataclassFactory
 from tests.models import Person
 
+try:
+    from sqlalchemy import __version__ as sqlalchemy_version
+except ImportError:
```

Comment:
Why is this catching of the `ImportError` needed?

Reply:
SQLAlchemy is an optional dependency, and didn't think the test suite should fail because it wasn't installed

It's still technically optional after the rebase, but will be pinned to V2 and above.

Happy to either do keep something similar after rebasing, or just remove the `skipif` check altogether!

Reply:
Aah the optional is for the users of `polyfactory`. It will still be installed as part of the development workflow since `poetry install --extras full` will include `sqlalchemy`. 

Yeah removing it is fine for now. We can deal with it when we add support for SQLA v1.